lib/restpack_serializer/serializable/side_load_data_builder.rb

```diff
@@ -18,7 +18,7 @@ def side_load_belongs_to
       def side_load_has_many
         has_association_relation do |options|
           if join_table = @association.options[:through]
-            options.scope = options.scope.joins(join_table).distinct
```

Comment:
It's not immediately obvious (it wasn't to me anyway), but specifying the table name is necessary to avoid ambiguity with the joined table.